src/content/docs/using-new-relic/cross-product-functions/install-configure/networks.mdx

```diff
@@ -35,11 +35,11 @@ For future updates to required and supported protocol versions, follow the [`Sec
 
 ## User-facing domains [#user-facing-domains]
 
-Update your allow list to ensure New Relic can communicate with a number of integral domains (listed below). Blocking domains can cause issues with individual product features or prevent pages from loading altogether.
+Update your allow list to ensure New Relic can communicate with a number of integral domains that are listed in this section. Blocking domains can cause issues with individual product features or prevent pages from loading altogether.
 
 This list doesn't cover domains that New Relic connects to that can be blocked without affecting your usage of the product. It also doesn't cover Nerdpacks or other features that communicate with external services that have additional domain requirements.
 
-If your organization uses a firewall that restricts outbound traffic, ensure that the following domains are added to the allow list:
+If your organization uses a firewall that restricts outbound traffic, follow specific procedures for the operating system and the firewall you use to add the following domains to the allow list.
```

Comment:
```suggestion
If your organization uses a firewall that restricts outbound traffic, follow the specific procedures for the operating system and the firewall you use to add the following domains to the allow list.
```

Reply:
This is a tiny suggestion, but I felt like "follow specific procedures" implied you this doc would provide them. I think simply adding "the" helps imply that the user has to track those down.

Reply:
makes sense, thanks! I will adjust that wording in the other spots too